dist/main.js

```diff
@@ -0,0 +1,62 @@
+const weatherApp = new WeatherAPP
+const renderer = new Renderer
+const background = document.querySelector('#background')
+
+if (navigator.geolocation) {
+  navigator.geolocation.getCurrentPosition((position) =>{
+     weatherApp.coords = { 
+         lat: position.coords.latitude,
+         lon: position.coords.longitude
+     }
+    
+     weatherApp.getCityByCoord().then(function (city) {
```

Comment:
maybe try to use async await since you used it everywhere else in your app. Might make it look cleaner